pkg/apiserver/apiserver.go

```diff
@@ -0,0 +1,68 @@
+package apiserver
+
+import (
+	"crypto/tls"
+	"errors"
+	"fmt"
+	"log"
+	"net"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+
+	pb "go.creack.net/telepilot/api/v1"
+	"go.creack.net/telepilot/pkg/jobmanager"
+)
+
+// Common errors.
```

Comment:
I _think_ ErrInvalidClientCerts is going to inherit this a godoc.

Reply:
FYI, this is how godoc looks like:

<img width="634" alt="image" src="https://github.com/user-attachments/assets/ce03d7f9-cdf0-42c1-a9f1-de48fd00be8e">

I believe it is clear enough, let me know if you think it would be better to add more details.

Reply:
That's good.